### PR TITLE
Feature#9 frontend login admin

### LINKED ISSUE #9: Frontend Login Admin
# REQUIREMENTS
- Login Form design

# DESIGN
image

### PR DESCRIPTION
**Pull request by Villaedi**
# Description
We create a new controller "login" and a view "admin" where we implement the login view styles.
Add the login view to the master branch

  ## Screenshots
Mockup
image
Frontend
127 0 0 1_3000_login_admin(iPad Mini)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.2'
 
-gem 'wdm', '>= 0.1.0'
+#gem 'wdm', '>= 0.1.0'
```
Comment (kevinnio) [inline Gemfile]: If we don't need this gem, why keep it? Let's get rid of it altogether. 👍

**Review by kevinnio — CHANGES_REQUESTED** [review]
Looks good to me, but I'd like to that gem removed instead of just commented out.

File: app/assets/stylesheets/src/layout/login.css.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+.form__input{
+    background-color: $background_color;
+    border: 0.1rem solid $secondary_color;
+    color: $font_color;
+    font-family: $font;
+    font-size: 2rem;
+    outline: none;
+    padding: 1rem; 
+}
```
Comment (YilKanda) [inline app/assets/stylesheets/src/layout/login.css.scss]: Check the indentation in this part.

File: app/views/login/admin.html.erb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
...2L52.08 11.62Q52.08 8.45 53.38 6.55Q54.67 4.66 57.12 4.66L57.12 4.66Q59.54 4.66 60.88 6.55Q62.21 8.45 62.21 11.62L62.21 11.62Q62.21 14.78 60.88 16.72Q59.54 18.65 57.12 18.65L57.12 18.65ZM57.12 16.15Q58.20 16.15 58.51 14.83Q58.82 13.51 58.82 11.59L58.82 11.59Q58.82 9.67 58.51 8.35Q58.20 7.03 57.12 7.03L57.12 7.03Q56.04 7.03 55.75 8.28Q55.46 9.53 55.46 11.33L55.46 11.33Q55.46 13.54 55.75 14.84Q56.04 16.15 57.12 16.15L57.12 16.15ZM64.49 18.48L64.49 5.11L67.85 4.82L67.85 7.01Q68.57 5.98 69.64 5.34Q70.70 4.70 72.36 4.70L72.36 4.70L72.36 7.99Q71.50 7.99 70.09 8.29Q68.69 8.59 67.85 9.34L67.85 9.34L67.85 18.19L64.49 18.48ZM74.18 22.54L74.18 5.11L77.54 4.82L77.54 6.14Q78.34 5.35 79.02 5.00Q79.70 4.66 80.62 4.66L80.62 4.66Q81.86 4.66 82.60 5.35Q83.33 6.05 83.65 7.52Q83.98 9.00 83.98 11.47L83.98 11.47Q83.98 14.86 83.20 16.75Q82.42 18.65 80.21 18.65L80.21 18.65Q79.20 18.65 78.60 18.26Q78 17.88 77.54 17.18L77.54 17.18L77.54 21.86L74.18 22.54ZM79.01 16.08Q79.80 16.08 80.20 15.14Q80.59 14.21 80.59 11.62L80.59 11.62Q80.59 9.14 80.20 8.11Q79.80 7.08 78.98 7.08L78.98 7.08Q78.14 7.08 77.54 8.02L77.54 8.02L77.54 15.00Q78.02 16.08 79.01 16.08L79.01 16.08Z" transform="translate(51, 5.332)"></path></svg>
```
Comment (kevinnio) [inline app/views/login/admin.html.erb]: This `SVG` here is distracting. What do you think about storing it in its own file (`app/assets/images/logo.svg`, for example), then insert it with `image_tag "logo.svg"`? Same goes for the other svgs in this view. 🤔

**Review by kevinnio — COMMENTED** [review]
Please move the SVGs out of the views, so the views are easier to read.

**Review by kevinnio — APPROVED** [review]

**Comment by DerekAyala** [comment]
@YilKanda could give us a code review ?

**Review by YilKanda — APPROVED** [review]
LGTM!

**Pull request merged**